manual_jb/content/introduction.md

```diff
@@ -39,6 +39,7 @@ Below you can find three Jupyter notebook [research examples](notebooks.md) that
 2. [Joplin testbed](notebooks/Joplin_testbed/Joplin_testbed.md) ([zip](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/Joplin_testbed.zip) file).
 3. [Lumberton testbed](notebooks/Lumberton_testbed/Lumberton_testbed.md) ([zip](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/Lumberton_testbed.zip) file).
 4. [Seaside testbed](notebooks/Seaside_testbed/Seaside_testbed.md) ([zip](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/Seaside_testbed.zip) file).
+5. [MMSA testbed](notebooks/MMSA_testbed/MMSA_Seismic_Functionality_Analysis_for_Interdependent_Buildings-Water-Power_using_Fragility_and_Repair_Rate_Curves.ipynb.md) ([zip](https://github.com/IN-CORE/incore-docs/blob/main/notebooks/MMSA_testbed.zip) file).
```

Comment:
You need to remove the .ipynb - link is broken on the docs build

Reply:
It should be:
(notebooks/MMSA_testbed/MMSA_Seismic_Functionality_Analysis_for_Interdependent_Buildings-Water-Power_using_Fragility_and_Repair_Rate_Curves.md)